src/Core/Form/IdentifiableObject/CommandBuilder/Product/SEOCommandBuilder.php

```diff
@@ -38,11 +38,12 @@ class SEOCommandBuilder implements ProductCommandBuilderInterface
 {
     public function buildCommand(ProductId $productId, array $formData): array
     {
-        if (!isset($formData['seo'])) {
+        if (!isset($formData['seo']) && !isset($formData['redirect_option'])) {
```

Comment:
```suggestion
        if (!isset($formData['seo'], $formData['redirect_option'])) {
```

Reply:
It doesn't work, the combined `isset` is like an && inside. In this case a `!a && !b` is equivalent to `!(a || b)` (basic boolean conversion)

Basically what I want here is either having seo data or redirection_option data (or both) But your modification forces to have both which is not the intented purpose (and unit tests confirm that it doesn't work ^^)

Reply:
Really ?

> If multiple parameters are supplied then isset() will return true only if all of the parameters are considered set. Evaluation goes from left to right and stops as soon as an unset variable is encountered.

Reply:
I think that's what he says, in this condition he wants to check that none are present, your condition will pass even if only one entry is not set.